app/assets/javascripts/sync.coffee.erb

```diff
@@ -7,6 +7,7 @@ $ = jQuery
   FAYE_HOST: "<%= Sync.server %>"
   PUSHER_API_KEY: "<%= Sync.api_key %>"
   CLIENT_ADAPTER: "<%= Sync.adapter %>"
+  ENCRYPTION_FLAG: "<%= Sync.encryption_flag %>"
```

Comment:
The adapter wasn't taking a boolean on my end. I'll send a follow-up if that's no longer the case.